### PR TITLE
Fixed the text center in pie box

### PR DESCRIPTION
**Pull request by YangQi007**

### CODE REVIEW

**Review by anderspitman — CHANGES_REQUESTED** [review]
@YangQi007 I think this should be closed in favor of other PRs right?

**Comment by YangQi007** [comment]
> @YangQi007 I think this should be closed in favor of other PRs right?

Sorry, I missed this comment. I think it was because of my email notification.

Yes, I think we can close this one. The misplaced text in the percent chart on the Safari will be solved in the iobio.viz repo.

**Comment by anderspitman** [comment]
@YangQi007 this should be good to go once you merge the latest version of iobio.viz.

**Review by anderspitman — APPROVED** [review]

**Pull request merged**
